src/app/apiAndObjects/api/diet/getUnmatchedTotals.ts

```diff
@@ -0,0 +1,48 @@
+/* tslint:disable: no-string-literal */
+import { HttpClient } from '@angular/common/http';
+import { DietDataSource } from '../../objects/dietDataSource';
+import { UnmatchedTotals } from '../../objects/unmatchedTotals';
+import { CacheableEndpoint } from '../../_lib_code/api/cacheableEndpoint.abstract';
+import { RequestMethod } from '../../_lib_code/api/requestMethod.enum';
+import { MnAvailabilityEndpointHelper } from './mnAvailabilityEndpointHelper';
+
+export class GetUnmatchedTotals extends CacheableEndpoint<
+  Array<UnmatchedTotals>,
```

Comment:
If you only wanted to return one UnmatchedTotals (which it looks like you do when you use it), you can change this from Array<UnmatchedTotals>to UnmatchedTotals (also makes 3rd param redundant), then use the "buildObjectFromResponse" (singular) method rather than "buildObjectsFromResponse".